test/jit/test_freezing.py

```diff
@@ -1559,3 +1560,33 @@ def forward(self, x):
         output_s = mod.forward(input)
         output_f = frozen_mod.forward(input)
         self.assertEqual(output_s, output_f)
+
+    @unittest.skipIf(not RUN_CUDA, "requires CUDA")
+    def test_freeze_conv_relu_fusion(self):
+        class Net(nn.Module):
```

Comment:
We're going to want to add more tests around bias being present/not present, and NCHW memory format, as well as Conv1d/Conv2d/Conv3d. This is a good example test: https://github.com/pytorch/pytorch/blob/master/test/jit/test_freezing.py#L1373.